tests/3d_examples/test_3d_dambreak/Dambreak.cpp

```diff
@@ -131,9 +131,9 @@ int main()
 	/** Output the body states for restart simulation. */
 	RestartIO restart_io(in_output, system.real_bodies_);
 	/** Output the mechanical energy of fluid body. */
-	RegressionTestEnsembleAveraged<BodyReducedQuantityRecording<TotalMechanicalEnergy>>
+	BodyReducedQuantityRecording<TotalMechanicalEnergy>
 		write_water_mechanical_energy(in_output, water_block, gravity);
-	RegressionTestDynamicTimeWarping<ObservedQuantityRecording<Real>>
+	ObservedQuantityRecording<Real>
```

Comment:
yes, this reverts the upstream changes.
So I would handle it with cmake flags.

Reply:
Done